cypress/integration/shared.js

```diff
@@ -192,6 +198,112 @@ export const waitForClusterInstallation = () => {
   cy.contains('div.pf-c-progress__description', 'Installed', { timeout: CLUSTER_CREATION_TIMEOUT });
 };
 
+export const loginOCM = (userName, password) => {
+  //Login to ocm
+  cy.visit('');
+  cy.get('#username').should('be.visible');
+  cy.get('#username').type(userName);
+  cy.get('#username').should('have.value', userName);
+  cy.get('#login-show-step2').click();
+  cy.get('#password').should('be.visible');
+  cy.get('#password').type(password);
+  cy.get('#password').should('have.value', password);
+  cy.get('#kc-form-login').submit();
+
+  // visit ocm environment specified by CYPRESS_HOST_ENV
+  switch (HOSTED_ENV) {
+    case 'staging':
+      cy.visit('');
+      break;
+    case 'integration':
+      cy.visit('', {
+        qs: {
+          env: 'integration',
+        },
+      });
+      break;
+    case 'production':
+      break;
+  }
+};
+
+export const makeApiCall = (apiPostfix, method, responseHandler, requestBody = {}) => {
```

Comment:
this makeApiCall is relevant only to OCM flow, so its name need to represent it somehow, i.e ocmMakeApiCall.